assets/js/accordeon.js

```diff
@@ -1,34 +1,34 @@
 /* ===== Zeste de Savoir ====================================================
    Accordeon for sidebar
    ---------------------------------
-   Author: Alex-D
+   Author: Alex-D, firm1
    ========================================================================== */
 
-(function($) {
-  'use strict'
+(function() {
+  function accordeon(element) {
+    const nextSearchElt = ['OL', 'UL']
 
-  function accordeon($elem) {
-    $('h4 + ul, h4 + ol', $elem).each(function() {
-      if (!$(this).hasClass('unfolded')) {
-        if ($('.current', $(this)).length === 0) { $(this).hide() }
-      }
-    })
+    Array.from(element.querySelectorAll('h4 + ul, h4 + ol'))
+      .filter(item => !item.classList.contains('unfolded'))
```

Comment:
On peut directement filtrer les classes dans le sélecteur, ce qui permet d'avoir un code un peu plus efficace.

```suggestion
    Array.from(element.querySelectorAll('h4 + ul:not(.unfolded), h4 + ol:not(.unfolded)'))
```